### ISSUE TITLE
Feature request: Add support to JSON:API as a schema type

### ISSUE THREAD

**Issue by swilgosz** [a85/POSTMan-Chrome-Extension#9165]
**Is your feature request related to a problem? Please describe.**

We are developing our API collections using [JSON:API](https://jsonapi.org) standard and we would love to create a new API in Postman for finished Services.

However, currently, we can only choose OpenAPI, RAML or GraphQL as schema types. There is no JSON:API nor even the custom one.

![image](https://user-images.githubusercontent.com/8088317/96036001-211d0300-0e64-11eb-82a0-314f2dc79a51.png)

**Describe the solution you'd like**

Ideally, add an option to choose JSON:API standard as a schema type

**Describe alternatives you've considered**

Delivering the `custom` option possibly could work.

**Additional context**

Basically, we cannot use the whole APIs feature with our chosen API schema.

![image](https://user-images.githubusercontent.com/8088317/96036185-66413500-0e64-11eb-8e76-ddeeb15d89a7.png)

**Comment by n2ygk** [comment]
JSON:API can be described by OpenAPI 3.0 -- well, with one [bug fix](https://github.com/OAI/OpenAPI-Specification/pull/2117) in the OpenAPI spec which is coming in the 3.1 release having to do with JSON:API requiring the DELETE method to have a request body.

For example, [Django-REST-Framework-JSON-API](https://django-rest-framework-json-api.readthedocs.io/en/stable/) release 4.0 now has an initial implementation of automatic JSON:API schema generation based on the serializer definitions. Please use (improve!) the [`components` boilerplate](https://github.com/django-json-api/django-rest-framework-json-api/blob/master/rest_framework_json_api/schemas/openapi.py#L19-L282) from that code as a start!

I've successfully imported an OpenAPI 3.0 spec generated by this code into Postman. There's one Postman bug (#9053) that appears to have been closed in error having to do with representing deepObject.

I hope this helps.
